selfservice/flow/registration/hook.go

```diff
@@ -172,9 +172,14 @@ func (e *HookExecutor) PostRegistrationHook(w http.ResponseWriter, r *http.Reque
 		WithField("identity_id", i.ID).
 		Info("A new identity has registered using self-service registration.")
 
-	trace.SpanFromContext(r.Context()).AddEvent(events.NewIdentityCreated(r.Context(), i.ID))
+	trace.SpanFromContext(r.Context()).AddEvent(events.NewRegistrationSucceeded(r.Context(), string(a.Type), a.Active.String()))
```

Comment:
Also here, we seem to have the provider ID now.

Reply:
done!